docs/src/dev-env.md

```
@@ -0,0 +1,79 @@
+- **kubectl:** Command-line tool for interacting with Kubernetes. Install it from [kubernetes.io](https://kubernetes.io/docs/tasks/tools/).
+- **Helm:** Package manager for Kubernetes. Install it from [helm.sh](https://helm.sh/docs/intro/install/).
+- **Docker Desktop/Kind/Minikube:** A local kubernetes cluster. Make sure you have the local cluster running before development.
+- **Make:** Helps in working with the project.
+- **Istio:** Required to test the project with istio. Install from [istio.io](https://istio.io/)
+- **k6:** Required to load test the project. Install from [k6.io](https://k6.io/)
+
+## 2. Clone the Repository
+
+Clone the KubeElasti repository from GitHub to your local machine:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)